DevOps/frankfurter/scripts/update/create_update.sh

```diff
@@ -23,6 +25,17 @@ fi
 mkdir -p $TMP_DIR
 echo "Done."
 
+if [ -d "$DOC_DIR" ]; then
+    echo "Cleaning old documentation ... "
+    rm -rf $DOC_DIR/*
```

Comment:
These `$VAR` variables should be quoted; otherwise, the script will break when paths with spaces are used.

Reply:
Thanks for the catch. Will correct this.